### ISSUE TITLE
Add rotation to AOE and ST when Dragonrage is not active

### ISSUE THREAD

**Issue by Dragonfo** [Hekili/hekili#2071]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### WoW Version

Retail (Dragonflight)

### Describe the Issue

If we are in M + in a raid and do not use Dragonrage, then the rotation does not work, or rather it does not show the special features and their priority. If we use Dragonrage and after it ends, the rotation will work correctly. But if we do not use Dragonrage and want to hold it for a very long period, then the rotation will offer us a not quite correct rotation. In AOE, it's even worse there, there will just be endless Azure Strike spam in 3+ targets.
Please, please add a condition so that you can use a full rotation without Dragonrage. Because we don't always want to use Dragonrage by cd.

### How to Reproduce

Start a fight, and don't use Dragonrage.

### Talent Loadout

BsbBnWR3R1Z1HmbUKIpvm3XXhTDCAAAAAAKaIJikEJkEJlkk0CSLkIhkEIJJSESSC

### Snapshot

https://pastebin.com/jGG94rJa

**Comment by Hekili** [comment]
Could you rephrase your ticket so I can understand better?  Answer these questions specifically:

What are you seeing that you shouldn't be seeing?

What do you think you should be seeing?

Saying "rotation will work correctly" or "not quite correct rotation" does not mean anything useful to me.  My best guess is that you should disable /hekili > Devastation > Toggles > Cooldown: Show Separately - Use Actual Cooldowns.

**Comment by Dragonfo** [comment]
Use rotation in ST without Fire Breath and Eternity Surge. 
Rotation in aoe can also be used without Fire Breath and Eternity Surge.
The rotation is now all built strictly under Dragonrage. And if you do not use Dragonrage cooldown, then the rotation will not work not in AOE not in ST. Because the addon will strictly adhere to the rule of using Dragonrage. It is enough just to make the optional use of Dragonrage so that there is a choice to use it by cooldown or not. 
I attached a screenshot to make it clearer what I mean.
![1](https://user-images.githubusercontent.com/63850853/205455123-20969d55-926a-4aca-9751-eafa2e88c342.jpg)
I crossed out those abilities that we do not want to use at the moment, but I want the rotation to continue after them as indicated in the picture with the same priorities, both in AOE and in ST. The conditions that are provided for by rotation worked. But if we activate our ability, then we work according to the usual scenario as shown below in the screenshot without already crossed out abilities. Dragonrage lacks OffGCD in rotation.
![2](https://user-images.githubusercontent.com/63850853/205455206-81db3f3d-8330-417b-89cd-07dcd8f892d0.jpg)

**Comment by Hekili** [comment]
You opted into that behavior by checking "Cooldown: Show Separately - Use Actual Cooldowns` in /hekili > Devastation > Toggles.  If you uncheck that, the other displays (Primary, AOE) will treat Dragonrage as though it's on cooldown when generating recommendations.

**Comment by Dragonfo** [comment]
I attached a Snapshot where it is clearly shown that only one Azure Strike recommendation occurs in AOE. If we are fighting with 3+ targets. Does not happen beyond the recommendation in AOE until we activate Dragonrage. You can personally verify this.

**Comment by Dragonfo** [comment]
https://pastebin.com/pK1WqNSd

**Comment by Hekili** [comment]
I answered the question.  Uncheck `Cooldown: Show Separately - Use Actual Cooldowns`.

**Comment by Dragonfo** [comment]
I take it you don 't want to understand this problem ? Do you suggest that I always use Dragonrage in any situation? Other classes I play for (DK, Hunter, War) you made an optional choice with the fact that the rotation is going normally without a burst, and here you just suggested that Dragonrage should always be used in any situation. Please read here what I have attached in the screenshot carefully.
![1](https://user-images.githubusercontent.com/63850853/205477089-eb6e071d-103e-4c3b-87cc-a176ef24a0ea.jpg)
I hope you can understand now that we can't use Dragonrage just like that, as you wrote.

**Comment by Hekili** [comment]
> I take it you don 't want to understand this problem ?

No, I do understand it.  Did you uncheck that setting or no?

> Do you suggest that I always use Dragonrage in any situation?

No, I suggest you use toggles to control when it is recommended and when it isn't.  That's what they're there for.  There's no "use Dragonrage at any time" recommendation; all of its entries have conditions.

For your screenshot, there's no way for the addon to know, especially immediately on pull, how long you can hold Dragonrage without risking losing a cast of Dragonrage over the course of a fight.  The addon can't know that your raid is going to pop Bloodlust 10 seconds into the pull, 40 seconds into the pull, or 9 minutes into the pull.

That is why toggles are implemented; you can control if the addon recommends it or not.

**Comment by Dragonfo** [comment]
Look, I turned off this setting, but then I have to play like this so that the entire rotation of 3 icons is shown.
![1](https://user-images.githubusercontent.com/63850853/205501681-6b359c82-9133-441e-bf04-0c5938d46d8b.jpg)
In this setup, the addon will still recommend me to use dragon rage. 
In order for me to control the rotation, I will need to increase the sequence of the rotation display, so as not to use rotation with dragon rage activated. 
Rotation is shown if we activate Dragon Rage. 
![1](https://user-images.githubusercontent.com/63850853/205501798-c74b84f6-96e2-4c63-b7f5-53febdb24964.jpg)
But the rotation is shown with the Show Separately option enabled.
![1](https://user-images.githubusercontent.com/63850853/205501925-0efd1f87-a3e7-4373-a14f-efa39090b05c.jpg)
I don't need the addon to calculate the time to BL. It's enough to simply return the rotation as it was in the prepatch when the dragons first appeared. Then everything worked as it should. I'm just trying to convey my idea that if we use dragon rage at the first recommendation of the addon, we will lose damage. Because as you correctly wrote above, the BL conditions may be different. But please, please add a small condition so that you can control the rotation yourself a little without taking into account the active dragon rage.

**Comment by Hekili** [comment]
No, you just need to also toggle Cooldowns off when you don't want Dragonrage to be recommended.

**Comment by Dragonfo** [comment]
I have completely disabled cooldowns. It turned out what I wanted to see. The only last question then, Dragon Rage and Tip the Scales, I'm not how it turns out now I can't track through the addon? Only now with the help of already additional WA ? Thank you for pushing for this method. I'm sorry if I didn't immediately understand your exact thought.
